packages/split-view/src/SplitView.ts

```diff
@@ -0,0 +1,324 @@
+                );
+                break;
+            case 'ArrowDown':
+                this.movePosition(
+                    event,
+                    !this.isLTR || this.vertical
+                        ? ARROW_KEY_CHANGE_VALUE
+                        : -1 * ARROW_KEY_CHANGE_VALUE
+                );
+                break;
+            case 'PageDown':
+                this.movePosition(
+                    event,
+                    !this.isLTR || this.vertical
+                        ? PAGEUPDOWN_KEY_CHANGE_VALUE
+                        : -1 * PAGEUPDOWN_KEY_CHANGE_VALUE
+                );
+                break;
+            case 'Home':
+                event.preventDefault();
+                this.updatePosition(this.minPos);
+                break;
+            case 'End':
+                event.preventDefault();
+                this.updatePosition(this.maxPos);
+                break;
+        }
+    }
+
+    protected resize(): void {
+        this.minPos = Math.max(this.primaryMin, this.size - this.secondaryMax);
+        this.maxPos = Math.min(
+            this.primaryMax,
+            this.size - Math.max(this.secondaryMin, this.getSplitterSize())
+        );
+        this.updatePosition(this.dividerPosition);
+    }
+
+    private updatePosition(x: number): void {
+        let pos = Math.max(this.minPos, Math.min(this.maxPos, x));
+        if (x < this.minPos) {
+            pos = this.minPos;
+        }
+        if (x > this.maxPos) {
+            pos = this.maxPos;
+        }
+        if (pos !== this.dividerPosition) {
+            this.dividerPosition = pos;
+        }
+        this.isResizedStart = this.dividerPosition === this.minPos;
+        this.isResizedEnd = this.dividerPosition === this.maxPos;
+        this.isCollapsedStart = this.dividerPosition === 0;
+        this.isCollapsedEnd =
+            this.dividerPosition >= this.size - this.getSplitterSize();
+    }
+
+    public getSplitterSize(): number {
+        const el = this.shadowRoot.querySelector('#splitter') as HTMLElement;
+        return (
+            Math.round(
+                parseFloat(
+                    window
+                        .getComputedStyle(el)
+                        .getPropertyValue(this.vertical ? 'height' : 'width')
+                )
+            ) || SPLITTERSIZE
+        );
+    }
+
+    protected firstUpdated(changed: PropertyValues): void {
+        super.firstUpdated(changed);
+        this.primaryDefault = this.primaryDefault
+            ? this.primaryDefault
+            : Math.round(this.size / 2);
+        this.dividerPosition = this.primaryDefault;
+        this.resize();
+    }
+
+    protected updated(changed: PropertyValues): void {
+        super.updated(changed);
+
+        if (changed.has('dividerPosition')) {
+            this.paneSlot.style.setProperty(
+                '--spectrum-split-view-first-pane-size',
+                `${this.dividerPosition}px`
```

Comment:
We need to round this number to whole numbers at some point in the process to prevent subpixel layout from bubbling up in the UI across various screen resolutions. See here the handle is no longer "centered": 
![image](https://user-images.githubusercontent.com/1156657/106910381-2af69a80-66cf-11eb-80a7-63e921e13feb.png)

Reply:
Fixed